### DISCUSSION TITLE
Is it possible for users to install only the full release version instead of a preview version by default?

### DISCUSSION THREAD

**Discussion by laicasaane** [openupm/openupm#5228] [General]
For example, when run this command
```
openupm add com.coffee.ui-particle
```

I expect it to install version `4.9.1` instead of `5.0.0-preview.5` by default.

Is there currently a way to specify this behaviour, either by a special semver or package manifest?

**Comment by favoyang** [comment]
Please follow-up https://github.com/openupm/openupm-cli/issues/395 and https://github.com/openupm/openupm-cli/issues/64
